frontend/src/components/WidgetContainerTitleGroup.js

```diff
@@ -43,6 +70,8 @@ WidgetContainerTitleGroup.defaultProps = {
   subtitle: '',
   showHeaderBorder: false,
   className: 'padding-3 ',
+  enableCheckboxes: false,
+  exportRows: () => {},
```

Comment:
This should maybe be exportRows: null. Otherwise this is a line that cannot be tested, since the function is impossible to call.

Reply:
Doesn't matter except for coverage

Reply:
Good point, will null.